.env

```
@@ -2,6 +2,6 @@
 BUILD_VARIANT=rcvr
 NEXT_PUBLIC_BUILD_VARIANT=$BUILD_VARIANT
 
-NEXT_PUBLIC_API_BASE=https://api.rcvr.app/
+NEXT_PUBLIC_API_BASE=https://rcvr-stripe-test.herokuapp.com/
```

Now that this is going live (with a feature flip), it can go back to the original URL.
```suggestion
NEXT_PUBLIC_API_BASE=https://api.rcvr.app/
```

---

oops...good catch